Building DAG of jobs...
Your conda installation is not configured to use strict channel priorities. This is however crucial for having robust and correct environments (for details, see https://conda-forge.org/docs/user/tipsandtricks.html). Please consider to configure strict priorities by executing 'conda config --set channel_priority strict'.
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job stats:
job            count
-----------  -------
run_script1        1
total              1

Select jobs to execute...

[Thu Oct 12 13:10:36 2023]
rule run_script1:
    input: data/input_vector.txt, data/input_pool2.txt, data/input_pool3.txt, data/input_pool4.txt, data/input_pool6.txt, data/input_pool5.txt, data/input_pool7.txt, data/input_pool5_2.txt, data/input_pool7_2.txt, data/result_240120_barcode_counts_table.txt, data/barcode_counts_table_170620_pool2.txt, data/barcode_counts_table_200720_pool3.txt, data/barcode_counts_table_170620_pool4.txt, data/barcode_counts_table_280720_pool6.txt, data/barcode_counts_table_20920_pool5.txt, data/barcode_counts_table_20920_pool7.txt, data/barcode_counts_table_20920_pool5.txt, data/barcode_counts_table_20920_pool7.txt, data/manifests_pool1.txt, data/manifest_pool2.txt, data/manifest_pool3.txt, data/manifest_pool4.txt, data/manifest_pool6.txt, data/manifest_pool5_1.txt, data/manifest_pool7_1.txt, data/manifest_pool5_2.txt, data/manifest_pool7_2.txt
    output: output/final_phenocall.txt
    jobid: 0
    reason: Missing output files: output/final_phenocall.txt
    resources: tmpdir=/var/folders/ty/spqw6k0n4dn1yvpgn_swvjm80000gn/T

Activating conda environment: .snakemake/conda/65dd274bf847fa589a55db189949edc6_
Activating conda environment: .snakemake/conda/65dd274bf847fa589a55db189949edc6_
[Thu Oct 12 13:12:47 2023]
Error in rule run_script1:
    jobid: 0
    input: data/input_vector.txt, data/input_pool2.txt, data/input_pool3.txt, data/input_pool4.txt, data/input_pool6.txt, data/input_pool5.txt, data/input_pool7.txt, data/input_pool5_2.txt, data/input_pool7_2.txt, data/result_240120_barcode_counts_table.txt, data/barcode_counts_table_170620_pool2.txt, data/barcode_counts_table_200720_pool3.txt, data/barcode_counts_table_170620_pool4.txt, data/barcode_counts_table_280720_pool6.txt, data/barcode_counts_table_20920_pool5.txt, data/barcode_counts_table_20920_pool7.txt, data/barcode_counts_table_20920_pool5.txt, data/barcode_counts_table_20920_pool7.txt, data/manifests_pool1.txt, data/manifest_pool2.txt, data/manifest_pool3.txt, data/manifest_pool4.txt, data/manifest_pool6.txt, data/manifest_pool5_1.txt, data/manifest_pool7_1.txt, data/manifest_pool5_2.txt, data/manifest_pool7_2.txt
    output: output/final_phenocall.txt
    conda-env: /Users/vpandey/projects/githubs/Fertility_screen_final/Fertility_screen/.snakemake/conda/65dd274bf847fa589a55db189949edc6_

RuleException:
CalledProcessError in file /Users/vpandey/projects/githubs/Fertility_screen_final/Fertility_screen/Snakefile, line 47:
Command 'source /Users/vpandey/opt/miniconda3/bin/activate '/Users/vpandey/projects/githubs/Fertility_screen_final/Fertility_screen/.snakemake/conda/65dd274bf847fa589a55db189949edc6_'; set -euo pipefail;  python /Users/vpandey/projects/githubs/Fertility_screen_final/Fertility_screen/.snakemake/scripts/tmp0_yn7nf7.combine_all_pool.py' returned non-zero exit status 1.
  File "/Users/vpandey/projects/githubs/Fertility_screen_final/Fertility_screen/Snakefile", line 47, in __rule_run_script1
  File "/Users/vpandey/opt/miniconda3/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-10-12T131034.975682.snakemake.log
